server/sonar-server/src/main/java/org/sonar/server/computation/task/projectanalysis/analysis/Project.java

```diff
@@ -77,15 +86,9 @@ public String toString() {
     sb.append("uuid='").append(uuid).append('\'');
     sb.append(", key='").append(key).append('\'');
     sb.append(", name='").append(name).append('\'');
+    sb.append(", description=").append(description == null ? null : '\'' + description + '\'');
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Add parentheses to make the operator precedence explicit. ([squid:S864](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS864))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&branch=feature%2Fsl%2FSONAR-10261%2Fbranch_analysis_does_not_update_name_description_nor_links&issues=AWH8Yt3ja5Ltb7kAT2N8&open=AWH8Yt3ja5Ltb7kAT2N8)

[ISSUE_KEY]: AWH8Yt3ja5Ltb7kAT2N8